src/elements/common/media-query/index.js

```diff
@@ -0,0 +1,2 @@
+export { default } from './useMedia';
```

Comment:
Let's skip default exports for this and use named ons

`import { useMedia, withMedia } from 'box-ui-elements/es/components/media-query'`